tests/manual-test-cases/Group19-VIC-Product/19-10-VIC-Vulnerability.md

```diff
@@ -0,0 +1,35 @@
+Test 19-10 VIC Vulnerability scanning
+=======
+
+# Purpose:
+To verify that the vulnerability scanning feature of VIC product works as expected
+
+# References:
+[1- VIC vulnerability scan](TBD - waiting on official docs to link)
+
+# Environment:
+This test requires that a vSphere server is running and available
+
+# Test Steps:
+1. Download and install the VIC OVA from here:
+[OVA Releases](https://console.cloud.google.com/storage/browser/vic-product-ova-releases/?project=eminent-nation-87317&authuser=1)
+2. Register Harbor as a registry with Admiral
+3. Populate the harbor instance with images that can be pulled
+4. Do a docker login/pull from the harbor instance
+5. Do a docker push to the harbor instance
+6. Login to Admiral UI, Go to ‘Administration->Projects->“Repositories” tab’ locate the specified repository
+7. Click on the arrow to expand the embedded tag list of the specified repository and then click on the vertical ellipsis to open the popup menu.
+8. Click on 'Scan' to start the vulnerability analysis process
+9. Verify that if error occurred, user can click on the 'View Log'
+10. Verify different scanning status "Not Scanned" "Queued" "Scanning" "Error" "Complete" - status are indicated in the 'Vulnerability' column
+11. Verify result once the scan has been completed - Red: High , Orange: Medium, Yellow: Low, Grey: Unknown and Green: No vulnerabilities
```

Comment:
My question was intended to address if we want to test each of the result types. If we do, then we should specify that an image for each status should be uploaded and scanned.